Define postMessage method

In Electron 9.1.0, I can't build tests using this library. Because `postMessage` method is defined in the latest Electron, but this library does not implement it.

```
    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
 
    spec/mock/electron.ts:6:21 - error TS2352: Conversion of type 'ipcRenderer' to type 'IpcRenderer' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
 
      Property 'postMessage' is missing in type 'ipcRenderer' but required in type 'IpcRenderer'.
 

 
    6 const ipcRenderer = mocked.ipcRenderer as IpcRenderer
 
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 

 
      node_modules/electron/electron.d.ts:4884:5
 
        4884     postMessage(channel: string, message: any, transfer?: MessagePort[]): void;
 
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
        'postMessage' is declared here.
 
````

--- a/example/jest/package.json
+++ b/example/jest/package.json
@@ -33,7 +33,7 @@
     }
   },
   "dependencies": {
-    "electron": "7.1.2",
+    "electron": "9.0.3",
     "electron-mock-ipc": "file:../../"
   },
   "devDependencies": {

--- a/example/mocha/package.json
+++ b/example/mocha/package.json
@@ -8,7 +8,7 @@
     "spec": "NODE_ENV=test mocha -r ts-node/register ./spec/**/*.spec.ts"
   },
   "dependencies": {
-    "electron": "7.1.2",
+    "electron": "9.0.3",
     "electron-mock-ipc": "file:../../"
   },
   "devDependencies": {

--- a/src/ipc-renderer.ts
+++ b/src/ipc-renderer.ts
@@ -37,7 +37,7 @@ class ipcRenderer implements IpcRenderer {
   }
 
   invoke(channel: string, ...args: any[]): Promise<any> {
-    return new Promise(resolve => {
+    return new Promise((resolve) => {
       this.emitter.once(channel, (_ev: IpcRendererEvent, ...args: any[]) => {
         resolve(...args)
       })
@@ -59,6 +59,10 @@ class ipcRenderer implements IpcRenderer {
 
   sendToHost(_channel: string, ...args: any[]): void {
     console.log(args)
+  }
+
+  postMessage(_channel: string, message: any, _transfer?: MessagePort[]): void {
+    console.log(message)
   }
 
   /**

